tests/onnx/data/reference_graphs/quantization/ssd-12.dot

```diff
@@ -1072,7 +1072,7 @@ strict digraph  {
 "310 Unsqueeze_345" -> "312 Concat_347"  [label="[1]", style=dashed];
```

Comment:
Why this model and others that didn't contain `model_type` in the configuration, were changed?

Reply:
NNCFGraphBuilder for ONNX was updated

Reply:
It was, but why the changes came to the models that shouldn't contain specific changes? Only the place of call `get_edge_shape` changed and some specific `ONNXExtendedLayerAttributes` were added. How it affects .dots?

Reply:
Discussed